Riot/Assets/en.lproj/Vector.strings

```diff
@@ -1060,9 +1060,9 @@
 // MARK: Self verification start
 
 // New login
-"device_verification_self_verify_alert_title" = "New Sign In";
-"device_verification_self_verify_alert_message" = "Use this session to verify your new one, granting it access to encrypted messages: %@\nIf you didn’t sign in to this session, your account may be compromised.";
-"device_verification_self_verify_alert_cancel_action" = "This wasn't me";
+"device_verification_self_verify_alert_title" = "New login. Was this you?";
+"device_verification_self_verify_alert_message" = "Verify the new login accessing your account: %@";
```

Comment:
Nad did not want to display the device id here. That is why he wrote in the issue:
"Verify the new login accessing your account & messages"